apps/full-site-editing/full-site-editing-plugin/blocks/post-content/edit.js

```diff
@@ -78,7 +78,11 @@ const PostContentEdit = compose(
 					>
 						<div className="post-content-block__selector">
 							<div>{ __( 'Select something to preview:' ) }</div>
-							<PostAutocomplete postType={ [ 'page', 'post' ] } onSelectPost={ onSelectPost } />
+							<PostAutocomplete
+								initialValue={ get( selectedPost, 'title.rendered' ) }
```

Comment:
we need to be passing `get()` paths as an array. by itself it's not a performance issue but in aggregate the string form is a costlier call because it has to take a code path that does string processing. this is also true in cases like `posts[ ${ postId } ]` as it's faster to pass `[ 'posts', postId ]`

```js
get( selectedPost, [ 'title', 'rendered' ] )
```

cc: @Automattic/lannister @Automattic/wp-lannister